backend/routes/meal.js

```diff
@@ -24,6 +33,18 @@ router.post('/api/meals', express.json(), async (req, res) => {
     res.json({ mealId });
 });
 
+/**
+ * Route for adding meal image.
+ * @param {Object} req - The request object.
+ * @param {Object} req.body - Request body.
+ * @param {Object} req.body.imageData - Image data.
```

Comment:
I think this is not necessarily true. Instead of saying that `req.body.imageData` contains the image data, `req.body` actually is the image data itself. This can be seen on the first line of the handler where we set `const imageData = req.body` instead of `const { imageData } = req.body`.